### DISCUSSION TITLE
如何实现任务栏系统控制条

### DISCUSSION THREAD

**Discussion by Aurora-Simba** [hooke007/mpv_PlayKit#387] [Q&A 问答] [answered]
这种
![屏幕截图 2023-11-11 122059](https://github.com/hooke007/MPV_lazy/assets/106955617/c486bd5e-52b7-4f95-a7f7-213919eee6d5)
![屏幕截图 2023-11-11 121907](https://github.com/hooke007/MPV_lazy/assets/106955617/13b8778e-dc06-4c39-aa4a-5e9b83eab0d9)

**Comment by zhongfly** [comment]
要软件自己支持吧…我是不清楚怎么用其他工具来实现

**Reply by Aurora-Simba** [reply]
那我去上游提issues了

**Reply by zhongfly** [reply]
我看了下正式名称应该是这个： thumbnail-toolbars
https://learn.microsoft.com/zh-cn/windows/win32/shell/taskbar-extensions#thumbnail-toolbars

**Reply by hooke007** [reply]
> 那我去上游提issues了

https://github.com/mpv-player/mpv/issues/5226

**Comment by hooke007** [comment, marked as answer]
https://github.com/qwerty12/mpv-taskbar-buttons
还有第三方的黑客方案
